{"uid":"ec7d7f669e45975f","name":"test_user_agent[drivers3-clients3-PythonSDK/2 (Python 1; Win; ciso) ConnectorA/0.1.1 ConnectorB/0.2.0]","fullName":"tests.unit.utils.test_usage_tracker#test_user_agent","historyId":"565c49dae9fc957ac3ad8819d17c4b7d","time":{"start":1730257957534,"stop":1730257957583,"duration":49},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7f3942d46490>\nfixturedef = <FixtureDef argname='clients' scope='function' baseid=''>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7f3943769c70>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"clear_cache","time":{"start":1730257957528,"stop":1730257957528,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"global_fake_fs","time":{"start":1730257957529,"stop":1730257957533,"duration":4},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"global_fake_fs::0","time":{"start":1730257957589,"stop":1730257957590,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.unit.utils"},{"name":"suite","value":"test_usage_tracker"},{"name":"host","value":"fv-az1247-605"},{"name":"thread","value":"2143-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.utils.test_usage_tracker"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"clients","value":"[]"},{"name":"drivers","value":"[('ConnectorA', '0.1.1'), ('ConnectorB', '0.2.0')]"},{"name":"expected_string","value":"'PythonSDK/2 (Python 1; Win; ciso) ConnectorA/0.1.1 ConnectorB/0.2.0'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":44,"skipped":0,"passed":58,"unknown":0,"total":102},"items":[{"uid":"7024d9c983aa0feb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/343//#testresult/7024d9c983aa0feb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730257822951,"stop":1730257823000,"duration":49}},{"uid":"c6234cb457274ae0","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/c6234cb457274ae0","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171586537,"stop":1730171586587,"duration":50}},{"uid":"37c52987bc98708a","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/342//#testresult/37c52987bc98708a","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171449960,"stop":1730171450009,"duration":49}},{"uid":"b5780001825e60ae","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/b5780001825e60ae","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085323603,"stop":1730085323653,"duration":50}},{"uid":"9189a98ac161cac6","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/341//#testresult/9189a98ac161cac6","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085169068,"stop":1730085169118,"duration":50}},{"uid":"d13edec857405438","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/d13edec857405438","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998868395,"stop":1729998868444,"duration":49}},{"uid":"bbee58827a63f365","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/340//#testresult/bbee58827a63f365","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998739897,"stop":1729998739947,"duration":50}},{"uid":"e5236b0887cde839","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/e5236b0887cde839","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729912175913,"stop":1729912175962,"duration":49}},{"uid":"bd0d7583bef97a23","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/bd0d7583bef97a23","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729912038297,"stop":1729912038347,"duration":50}},{"uid":"a6e296ffc3ca37eb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/335//#testresult/a6e296ffc3ca37eb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825991796,"stop":1729825991845,"duration":49}},{"uid":"35e158ea6b64ff6c","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/35e158ea6b64ff6c","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825852578,"stop":1729825852627,"duration":49}},{"uid":"3711e6fd29edbbf0","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/334//#testresult/3711e6fd29edbbf0","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739486837,"stop":1729739486887,"duration":50}},{"uid":"e5100162c6a0c898","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/e5100162c6a0c898","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739334843,"stop":1729739334891,"duration":48}},{"uid":"9f44a0ecf1c2ac3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/333//#testresult/9f44a0ecf1c2ac3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729653094948,"stop":1729653094997,"duration":49}},{"uid":"e03bca9df49b2fb3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/e03bca9df49b2fb3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729652954132,"stop":1729652954181,"duration":49}},{"uid":"64fb676e376897f4","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/332//#testresult/64fb676e376897f4","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729566745641,"stop":1729566745690,"duration":49}},{"uid":"f480ba29080926b3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/335//#testresult/f480ba29080926b3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729566609104,"stop":1729566609153,"duration":49}},{"uid":"6d5132994332fe7b","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/331//#testresult/6d5132994332fe7b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729480453414,"stop":1729480453461,"duration":47}},{"uid":"5517cee3ef508759","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/334//#testresult/5517cee3ef508759","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729480327582,"stop":1729480327632,"duration":50}},{"uid":"90887844e8fc5cfb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/330//#testresult/90887844e8fc5cfb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729394105660,"stop":1729394105708,"duration":48}}]},"tags":[]},"source":"ec7d7f669e45975f.json","parameterValues":["[]","[('ConnectorA', '0.1.1'), ('ConnectorB', '0.2.0')]","'PythonSDK/2 (Python 1; Win; ciso) ConnectorA/0.1.1 ConnectorB/0.2.0'"]}